blog/md/ref/big_data/hadoop/README.md

```diff
@@ -0,0 +1,191 @@
+# Hadoop
+
+### Step 1: Update Dockerfile
+
+```dockerfile
+# Dockerfile
+FROM ubuntu:22.04 AS builder
+
+# Install Java Development Kit
+RUN apt-get update && apt-get install -y openjdk-8-jdk
+
+# Set the working directory
+WORKDIR /app
+
+# Add HelloWorld.java to the working directory
+ADD HelloWorld.java .
+
+# Compile the Java program
+RUN javac -source 8 -target 8 HelloWorld.java
+
+# Use a smaller base image for the final image
+FROM ubuntu:22.04
```

Comment:
@Thoophoms , Trisha, there are two From here in this docker file, could you please check if it is correct?